vendor/github.com/OpenBazaar/spvwallet/blockchain.go

```diff
@@ -299,7 +293,7 @@ func (b *Blockchain) GetBlockLocatorHashes() []*chainhash.Hash {
 		}
 		start += 1
 	}
-	return ret
+	return blockchain.BlockLocator(ret)
```

Comment:
Where does this get used? Looks like it's exposing a large abstraction outside of the wallet and I'm curious what future dependencies this creates on the spvwallet interface.